tests/integration/test_storage.sh

```diff
@@ -69,20 +69,20 @@ trap teardown EXIT
 # If /etc/sysconfig/docker-storage-setup is missing, atomic should create the file.
 
 rm -f /etc/sysconfig/docker-storage-setup
-OUTPUT=$(${ATOMIC} storage modify --add-device $TEST_DEV_1 2>&1)
+if [ -z "$VGROUP" ];then
+   VGROUP="test-atomic-storage"
+   OUTPUT=$(${ATOMIC} storage modify --add-device $TEST_DEV_1 --vgroup $VGROUP 2>&1)
```

Comment:
> The volume group (VG) is being set here first time. why would it require a reset first ?

The error from docker-storage-setup is
```
ERROR: Docker has been previously configured for use with devicemapper graph driver. Not creating a new thin pool as existing docker metadata will fail to work with it. Manual cleanup is required before this will succeed.
INFO: Docker state can be reset by stopping docker and by removing /var/lib/docker directory. This will destroy existing docker images and containers and all the docker metadata.
```

This is what `atomic storage reset` will do.

The reset is necesarry because there is already a docker storage pool, and docker-storage-setup can not move it seamlessly into a different volume group.

Whe I initially wrote the tests I couldn't do any reset since the tests couldn't be destructive back then.  Can we do it now, though?  The reset will remove the atomic-test-* images which are used by other integration tests.